extensions/narayana-lra/runtime/pom.xml

```diff
@@ -16,11 +16,7 @@
     <dependencies>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy-jackson</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-rest-client</artifactId>
+            <artifactId>quarkus-rest-client-reactive</artifactId>
```

Comment:
won't a dependency on quarkus-rest-client-reactive here mean that it won't work with the classic one?

Reply:
Won't work with what?

Reply:
`rest-client-jackson`

Reply:
sorry, `quarkus-resteasy-jackson`

Reply:
It works just fine, that's what the integration test actually tests :)

The reactive rest client does **not** need the RESTEasy Reactive to work.

Reply:
I know it doesn't but I thought there may be some conflicts. Cool :)

Reply:
Nope, it's all good :)